package.json

```diff
@@ -139,7 +143,7 @@
     "deepmerge": "^1.2.0",
     "draft-convert": "^2.1.2",
     "draft-js": "^0.10.4",
-    "draft-js-alignment-plugin": "^2.0.3",
```

Comment:
According to our yarn.lock we're actually on `2.0.6`. Changelog is here: https://github.com/draft-js-plugins/draft-js-plugins/blob/master/packages/alignment/CHANGELOG.md ; the bugfix in 2.0.5 might be something that matters.